fresh_connection.gemspec

```diff
@@ -15,8 +15,8 @@ Gem::Specification.new do |spec|
   spec.license       = "MIT"
 
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
-  spec.bindir        = "exe"
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.bindir        = "bin"
```

Comment:
Yes, I know.  This is why I made that change -- to have any installed scripts put into the local `bin` directory.